CMakeLists.txt

```diff
@@ -18,6 +18,13 @@ if(NOT BUILD_SHARED_LIBS)
   set(LOG4CXX_COMPILE_DEFINITIONS LOG4CXX_STATIC)
 endif()
 
+option(EXPORT_ONLY_LOG4CXX_SYMBOLS "Only exports the log4cxx symbols" ON)
```

Comment:
What's the benefit of enabling this by default? Especially if it's unclear currently if different compilers are supported.

Reply:
I put the default to false.